testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -108,7 +93,7 @@ def parse_args():
         description="This script reads the open qam-manager requests and creates a json file that can be fed in the "
                     "BV testsuite pipeline")
     parser.add_argument("-v", "--version", dest="version",
-                        help="Version of SUMA you want to run this script for, options are 42 for 4.2 or 43 for 4.3",
+                        help="Version of SUMA you want to run this script for, options are 43 for 4.3",
```

Comment:
maybe change temporarily the message? 
```suggestion
                        help="Version of SUMA you want to run this script for, the only option for now is 43 for 4.3",
```

Reply:
I say temporarily, I mean until 5.0 comes out.

Reply:
Yup, good idea

Reply:
Done